presqt/api_v1/utilities/io/remove_path_contents.py

```diff
@@ -0,0 +1,21 @@
+import os
+import shutil
+
+
+def remove_path_contents(path, *file_exceptions):
```

Comment:
Could file_exceptions just a be list? Curious why you used the *args convention here.

Reply:
Oh... by the way, let me show type hinting in Python 3!